tooling/cli/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "cli.js": {
     "version": "1.0.0-rc.9",
-    "node": ">= 10"
+    "node": ">= 10.0.0"
```

Comment:
@lucasfernog I am not sure if this value is read anywhere else, so I am double checking if its fine to change it.

Reply:
all good